xbmc/cores/dvdplayer/DVDCodecs/Overlay/DVDOverlayCodecFFmpeg.cpp

```diff
@@ -237,10 +239,28 @@ CDVDOverlay* CDVDOverlayCodecFFmpeg::GetOverlay()
     if(m_Subtitle.rects[m_SubtitleIndex] == NULL)
       return NULL;
 
-    AVSubtitleRect& rect = *m_Subtitle.rects[m_SubtitleIndex];
+    AVSubtitleRect rect = *m_Subtitle.rects[m_SubtitleIndex];
     if (rect.pict.data[0] == NULL)
       return NULL;
 
+    m_height = m_pCodecContext->height;
+    m_width  = m_pCodecContext->width;
+
+    RENDER_STEREO_MODE render_stereo_mode = g_graphicsContext.GetStereoMode();
+    if (render_stereo_mode != RENDER_STEREO_MODE_OFF)
```

Comment:
I am not very experienced with 3d. Do we need to distinguish between 3D subtitles and 2D subtitles in 3D content?